src/theme.js

```diff
@@ -62,7 +62,21 @@ const darkPurple = '#407'
 const flatten = (name, colors) =>
   colors.reduce((a, b, i) => {
     const color = {
-      [name + i]: b
+      [name + i]: {
+        enumerable: true,
+        get() {
```

Comment:
Personally, I think this would be better handled as a custom prop type on the components that use `color` and `bg` props. The custom propTypes could go into their own module to reuse across multiple components, see this for an example: https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Reply:
What when consumed from a `theme`?

Reply:
Good point. I think we could almost consider consuming directly from the theme a *private API* and not part of the supported API – hopefully no one's using the old color names from the theme 😬

Reply:
A lot of people do :/ I have seen that a lot, and that's why a warning would hopefully make people do the change, at least for now, until we deprecate all the numbered colors.

Reply:
Good to know. I'd say if you see it in the wild, try to leave comments or make changes to use the new color names where possible